### PR TITLE
Initial updates to Readme

### PR DESCRIPTION
**Pull request by kedark3**
## Purpose

Since we are attempting to make this repo a collections repo, we need to also update readme to reflect new guidelines and information.

### CODE REVIEW

**Review by ehelms — APPROVED** [review]
Will wait for @evgeni to weigh in as well

Related to this, perhaps we should rename the repository to reflect its more than just roles (e.g. satellite-ansible-assets)

**Review by aoyawale — APPROVED** [review]
Everything looks good to me.

File: README.md
Comment (evgeni) [inline README.md]:
```suggestion
Ansible Roles, Playbooks (and Tower Configuration metadata) for installing, configuring and interacting with the Red Hat Satellite. The roles will be created using [foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) (FAM) where possible.
```

File: README.md
Comment (evgeni) [inline README.md]: Today it's 2.7+ upstream (see https://docs.ansible.com/ansible/latest/reference_appendices/release_and_maintenance.html), but 2.7 does not support collections and we've been not officially supporting it in FAM for a while now because of that.

I'd say 2.8+ is a good target, but have no problem if we want to say 2.9 is minimum either.

File: README.md
Comment (evgeni) [inline README.md]:
```suggestion
* The roles should be compatible with any Python supported by Ansible upstream (2.7, 3.5, 3.6, 3.7 at the time of writing), however, roles usually don’t require any special Python features (in contrast to modules).
```

Given 2.7 is the default Python on RHEL7…

File: README.md
Comment (evgeni) [inline README.md:22]: +"unless there is no other solution"? We don't have 100% coverage with FAM, so some things are only possible with hammer *today*.

File: README.md
Comment (evgeni) [inline README.md]: I have a hard time calling this a collection (in the ansible context) while not uploading it to galaxy, but I think we agreed that this is a staging/devel place which we don't want to publish to Galaxy.

Mind you, it's just about the work collection as it has a very special meaning in Ansible land.

On the other hand, I think collection is what we're going to use when consuming this in Ansible, so maybe it's OK?

**Review by evgeni — CHANGES_REQUESTED** [review]

File: README.md
Comment (aoyawale) [reply README.md:22]: I think at that point its fine since its an initial commit and we can add and document times we use those modules.

File: README.md
Comment (ehelms) [reply README.md]: I think whatever we go with we should rename the repository to match:

 * Collection
 * Assets
 * Junk drawer
 * Stuff
 * Compilation
 * Accumulation

Comment (aoyawale) [reply README.md]: yeah I think we can call it collection, make changes, make it better and later add it to galaxy. It doesn't have to be right away.

File: README.md
Comment (kedark3) [reply README.md]: Understood. I think we will stick to 2.9 then. I recall others on the call were ok with that being minimum as well. Thanks.

File: README.md
Comment (kedark3) [reply README.md]: Got it. Will add it. Thanks.

File: README.md
Comment (kedark3) [reply README.md:22]: Agreed. We want to avoid it where we can. But if no other solution, then shell etc. modules are ok. That's what we intend to say in the docs.

File: README.md
Comment (kedark3) [reply README.md]: I'm happy if @ehelms or @evgeni can rename the repo. Thanks.

Comment (evgeni) [reply README.md]: I like compilation, but fine with either.

If we want to rename (which sounds like a good idea), it can't be collection, as satellite-ansible-collection already exists.

Comment (cojmckee) [reply README.md]: I agree with @evgeni  about the name to be Compilation

Comment (ehelms) [reply README.md]: Three votes for compilation... let's roll with that as naming can always be a long conversation and until we are sure how we want to package this up (as a whole or split it up into logical collections) let's avoid 'collection' given that is a "branded" term in Ansible land.

Comment (kedark3) [reply README.md]: :+1:

**Comment by kedark3** [comment]
I will merge this PR if there are no objections by EOD today. Thank you.

File: README.md
Comment (ehelms) [reply README.md]: @kedark3 Can you update the wording here before merging from Collection to Compilation?

Comment (kedark3) [reply README.md]: @ehelms will do thanks :)

Comment (kedark3) [reply README.md]: Updated. There are instances in Readme that say it is a collection which I think is still correct.

**Pull request merged**
